docs/source/docker/advanced/datapool.rst

```diff
@@ -0,0 +1,102 @@
+Datapool
+=================
+
+The Lightly Datapool is a tool which allows users to incrementally build up a 
+dataset for their project. It keeps track of the representations of previously
+selected samples and uses this information to pick new samples in order to
+maximize the quality of the final dataset. It also allows for combining two 
+different datasets into one.
+
+- | If you're interested in how the datapool works, go to
+  | --> `How It Works`_
+
+- | To see how you can use the datapool, check out
+  | --> `Usage`_
+
+
+How It Works
+---------------
+
+The Lightly Datapool keeps track of the selected samples in a csv file called
+`datapool_latest.csv`. It contains the filenames of the selected images, their
+embeddings, and their weak labels. Additionally, after training a self-supervised
+model, the datapool contains the checkpoint `checkpoint_latest.ckpt` which was 
+used to generate the embeddings.
+
+The datapool is located in the `shared` directory. In general, it is a directory
+with the following structure:
+
+
+.. code-block:: bash
+
+    # example of a datapool
+    datapool/
+    +--- datapool_latest.csv
+    +--- checkpoint_latest.ckpt
+    +--- history/
+  
+The files `datapool_latest.csv` and `checkpoint_latest.csv` are updated after every
+run of the Lightly Docker. The history folder contains the previous versions of 
+the datapool. This feature is meant to prevent accidental overrides and can be 
+deactivated from the command-line (see `Usage`_ for more information).
+
+Usage
+---------------
+
+.. note:: To use the datapool feature, the Lightly Docker requires write access
```

Comment:
I would move the `note` blocks after the text. A power user just gets the information. A first time user will see the note block after reading through the info. I would prefer that pattern.